worker-macros/src/event.rs

```diff
@@ -51,7 +51,7 @@ pub fn expand_macro(attr: TokenStream, item: TokenStream, http: bool) -> TokenSt
             input_fn.sig.ident = input_fn_ident.clone();
 
             let error_handling = if respond_with_errors {
-                quote! { ::worker::Response::error(e.to_string(), 500).unwrap().into() }
+                quote! { ::worker::Response::error(format!("{}", e), 500).unwrap().into() }
```

Comment:
Just FYI, this is equivalent to the previous code. `ToString` is implemented for all types that implement `Display`: https://doc.rust-lang.org/std/string/trait.ToString.html